app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, file, directory = command.split()
+    if command == "mv":
+        command_ls = directory.split("/")
+        for direction in command_ls[:-1]:
+            file_directory = directory[: directory.index(direction)]
+            os.mkdir(f"{file_directory}{direction}")
```

Comment:
make a check for " / " for this part of code, it won't work correctly with command without " / "